src/schema/rules/sidecars/mri.yaml

```diff
@@ -77,8 +77,7 @@ ASLMRISequenceSpecifics:
 
 MTParameters:
     selectors:
-    - '"MTState" in sidecar'
-    - sidecar.MTState == "True"
+    - sidecar.MTState == true
```

Comment:
Why did you drop `MTState in sidecar`?

Reply:
Per Zoom discussion, this seems to be the consensus. We can assume that inspecting a field that doesn't exist won't raise an error, but will return a value that is distinct from False (e.g., None for Python). The validator implementation will just have to take that into account.